### DISCUSSION TITLE
The Future of Open Source

### DISCUSSION THREAD

**Discussion by Ryzen-1fd** [rubenlagus/TelegramBots#1419] [Q&A] [answered]
why peoples not like upload open source

**Comment by Nuwan-Srimal** [comment, marked as answer]
- Developers might be concerned about receiving negative feedback or harsh criticism on their code, especially if they feel it isn’t perfect.

- Some developers prefer to keep their work private, especially if they’re using it for personal projects or learning purposes.

**Comment by Ryzen-1fd** [comment]
- Developers might be concerned about receiving negative feedback or harsh criticism on their code, especially if they feel it isn’t perfect.

- Some developers prefer to keep their work private, especially if they’re using it for personal projects or learning purposes.
